pkg/rbac/parser.go

```diff
@@ -93,6 +93,18 @@ func (r *Rule) key() ruleKey {
 	}
 }
 
+func (r *Rule) KeyWithoutResources() string {
```

Comment:
Not sure if the name is ideal. It also adds verbs compared to the key

Reply:
Maybe let's enumerate what is in the key (KeyWith...)? (+unexport the func if possible)